src/main/java/com/powsybl/openloadflow/network/impl/LfBusImpl.java

```diff
@@ -169,23 +186,24 @@ void addBattery(Battery battery) {
     private void add(LfGenerator generator, boolean voltageControl, double targetV, double targetQ,
                      LfNetworkLoadingReport report) {
         generators.add(generator);
-        boolean voltageControl2 = voltageControl;
+        boolean fixedVoltageControl = voltageControl;
```

Comment:
What do you mean by "fixed" ?

Reply:
I mean repaired, because the input value is modified depending of additiona checks.
Maybe modifiedVoltageControl would be better?

Reply:
Yes "modifiedVoltageControl" is better.

Reply:
Done